test/PerformanceTests/SerializationBaselineTests/SerializationBenchmarks.cs

```diff
@@ -18,24 +18,21 @@ public class SerializationBenchmarks
         IExperimentWriter odataSyncCharPoolWriter;
         IEnumerable<Customer> data;
 
-        [Params(1000, 5000, 10000)]
-        public int dataSize;
+        public int dataSize = 5000;

```

Comment:
I set this to a fixed size instead of having different data sizes because the tests are linear. Increasing the data size increases the run time linearly which did not seem to provide any additional insight to justify the additional time spent running the benchmarks.

Reply:
Could this be reduced to 2500 and still give the same depth?